app/assets/stylesheets/components/_form.scss

```
@@ -139,3 +139,18 @@
         margin: 0 40px 0 40px;
     }
 }
+.form{
+    background-color: $tertiary-color-light;
+    display: flex;
+    flex-direction: row;
+    height: 60%;
+    margin: auto;
+    width: 90%;
+    border-radius: 10px;
+    padding: 10px;

```

ORder alphabetically 
```suggestion
    background-color: $tertiary-color-light;
    border-radius: 10px;
    display: flex;
    flex-direction: row;
    height: 60%;
    margin: auto;
    padding: 10px;
    width: 90%;
```

---

done :)